### PR TITLE
Adding Makefile and README using terraform docs

### PR DESCRIPTION
**Pull request by sylviamclaughlin**
# Summary | Résumé

Adding Makefile and READMEs generated by Terraform docs for all the modules in the LZ repo.

### CODE REVIEW

**Review by gcharest — APPROVED** [review]

File: terragrunt/aft/main/Makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.PHONY: fmt docs test
+
+fmt:
+	@terraform fmt -recursive
+
+docs:
+	@rm -rf .terraform
+	@rm -f .terraform.lock.hcl
```
Comment (patheard) [inline terragrunt/aft/main/Makefile:8]: Could you please remove these two lines from the Makefiles as well.

File: terragrunt/aft/main/Makefile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.PHONY: fmt docs test
```
[the remark this answers is not in the document]
Comment (sylviamclaughlin) [reply terragrunt/aft/main/Makefile]: Sorry! Should have realized it was there. It has been removed for all files.

File: terragrunt/aft/main/Makefile (hunk 1 above)
Comment (sylviamclaughlin) [reply terragrunt/aft/main/Makefile]: Removed for all files - thank you for flagging this.

File: terragrunt/aft/main/Makefile (hunk 2 above)
[the remark this answers is not in the document]
Comment (patheard) [reply terragrunt/aft/main/Makefile]: Thank you but we'll still need the [phony targets](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html) for the other two like so `.PHONY: fmt docs`

Sorry for not being clearer!

[the remark this answers is not in the document]
Comment (sylviamclaughlin) [reply terragrunt/aft/main/Makefile]: Sorry Pat! I guess I never have used phony targets so I did not understand what you meant. Now that I read about them, this is really useful to have in every Makefile! I have added them and hopefully, I got it all right this time. Thank you for all the suggestions.

**Review by patheard — APPROVED** [review]

File: terragrunt/aft/main/Makefile (hunk 2 above)
[the remark this answers is not in the document]
Comment (patheard) [reply terragrunt/aft/main/Makefile]: Not a problem at all - what you've got now looks perfect

**Pull request merged**